canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.jsx

```diff
@@ -25,6 +25,7 @@ import { RadioButton } from "carbon-components-react";
 import classNames from "classnames";
 import { STATES } from "./../../constants/constants.js";
```

Comment:
You can move `UPDATE_TYPE` here

Reply:
<img width="602" alt="image" src="https://user-images.githubusercontent.com/89609176/185458378-4f5fcc44-0b3f-45f5-a2ca-607248cf1ecf.png">